ompi/mca/osc/rdma/osc_rdma_passive_target.c

```diff
@@ -309,6 +442,53 @@ int ompi_osc_rdma_unlock_atomic (int target, ompi_win_t *win)
     return ret;
 }
 
+int ompi_osc_rdma_iunlock_atomic (int target, ompi_win_t *win)
```

Comment:
I think would be better called `tryunlock` rather than `iunlock` because it attempts to take a lock but does not return a request that can be waited on or tested until completion of the locking operation. If it fails to obtain the lock, it just returns a non-success value - the calling code must call this function again (rather than calling some sort of wait or test function).

Reply:
iunlock -> tryunlock